src/FFmpegReader.cpp

```diff
@@ -638,6 +650,11 @@ void FFmpegReader::Close() {
 	}
 }
 
+bool FFmpegReader::HasAlbumArt() {
+	return pFormatCtx && videoStream >= 0 && pFormatCtx->streams[videoStream]
+		&& (pFormatCtx->streams[videoStream]->disposition & AV_DISPOSITION_ATTACHED_PIC);
```

Comment:
I looked at the FFmpeg codebase. It seems like some formats such as `.mov`, `mkv`, `wtv`, `asf` may contain streams with `AV_DISPOSITION_ATTACHED_PIC` flag, but I think I need to test these cases.
> system-generated still frame image

That seems to irrelevant, though.

Reply:
> > system-generated still frame image
> 
> That seems to irrelevant, though.

Oh, yeah, sorry, it is. To this PR, I mean. System thumbnails are managed external to the media files.

Reply:
@PhysSong

If you need a test file, you can use `mkvpropedit` to add a PNG attachment to an MKV file:
```sh
mkvpropedit Video.mkv --add-attachment Image.png
```

Reply:
`mkvpropedit` is part of the [MKVtoolnix suite](https://mkvtoolnix.download/), BTW.

Reply:
I tested this with the step you provided. Since libopenshot takes the first video stream, `HasAlbumArt()` will check if that one is a static image. `ffprobe` says the video cover art is the second video stream, meaning that libopenshot won't recognize the image as album art.

Reply:
I'll add a comment to explain how this function works.